crates/wasmtime/src/runtime/vm/instance/allocator/pooling.rs

```diff
@@ -372,8 +382,12 @@ impl PoolingInstanceAllocator {
             live_tables: AtomicUsize::new(0),
             #[cfg(feature = "gc")]
             gc_heaps: GcHeapPool::new(config)?,
+            #[cfg(feature = "gc")]
+            live_gc_heaps: AtomicUsize::new(0),
             #[cfg(feature = "async")]
             stacks: StackPool::new(config)?,
+            #[cfg(feature = "async")]
+            live_stacks: AtomicUsize::new(0),
```

Comment:
I assume the cfg feature here vs at the use site is causing the current CI breakage